endToEndTests/test/queries/manySequences.json

```diff
@@ -0,0 +1,44 @@
+{
```

Comment:
Can we rename this file to fasta_manySequences? Then it will be next to the other fastaAligned test.